laplace-essay-on-probabilities.md

```diff
@@ -297,26 +297,26 @@ the black balls; and the probability of this case is evidently 1/3.
 
 We see by this example the influence of past events on the probability
 of future ones. Because the probability to extract a white ball from
-urn B, which primitively is 2/3, becomes 1/2 once we have extracted a
+urn B, which initially is 2/3, becomes 1/2 once we have extracted a
 white ball from urn C: it would be changed to certainty if we had
 extracted a black ball from the same urn. We will determine this
 influence by means of the following principle, which is a corollary of
 the preceding one.
 
 **5th Principle.** If we calculate *a priori* the probability of the
 event observed, and the probability of an event composed of that one
-and of another which we await; the second probability, divided by the
-first, will be the probability of the awaited event, given the
+and of another which we expect; the second probability, divided by the
+first, will be the probability of the expected event, given the
 observed event.
 
-Here is presented the question agitated[XXX] by some philosophers,
-touching the influence of the past on the probability of the
-future. Let us suppose that in the game of *cross* or *pillar*,
-*cross* has happened more often than *pillar*: by that alone, we will
+Here occurs the debate stirred by a few philosophers,
+touching upon the influence of the past on the probability of the
+future. Let us suppose that in the game of *heads* or *tails*,
+*heads* has happened more often than *tails*: by that alone, we will
 be brought to believe that in the makeup of the coin there is a cause
 constantly favoring this. So too, in the conduct of life, is
-consistent good fortune a proof of shrewdness, which the fortunate
-must employ preferentially. But if by the instability of circumstances
+consistent good fortune a proof of shrewdness, which must lead us to preferably employ happy people.
```

Comment:
It seemed more likely to me he was putting the subject last in that clause, than offering hiring advice, but on reread I clearly got it wrong. (https://fr.wikisource.org/wiki/Essai_philosophique_sur_les_probabilit%C3%A9s/1b) Still, might "lucky/fortunate" be more proper here than "happy" for "les personnes heureuses"? (I'm happy to defer to you -- just want to learn.)

Reply:
I agree, this is clearly heureux as in fortunate, but also I think maybe a reference to the proverb "heureux au jeu, malheureux en amour?"

Reply:
Something like "lead us to prefer employing the fortunate" might read better?

Reply:
did ^